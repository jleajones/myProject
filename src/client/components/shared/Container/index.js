import React from 'react';
import PropTypes from 'prop-types';

/**
 *
 * @param children
 * @returns {*}
 * @constructor
 */
const Container = ({ children }) => <div>{children}</div>;

Container.propTypes = {
  children: PropTypes.node.isRequired
};

// const MyContainer = styled.div`
//   padding: 0 20px;
//
//   @media (max-width: 600px) {
//     padding: 0;
//   }
// `;

export default Container;
