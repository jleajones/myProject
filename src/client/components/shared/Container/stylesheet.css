.container {
  padding: 0 20px;
}
@media only screen and (max-width: 600px) {
  .container {
    padding: 0;
  }
}
