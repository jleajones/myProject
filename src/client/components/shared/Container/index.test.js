import React from 'react';
import { shallow } from 'enzyme';
import Container from '.';

describe('<Container />', () => {
  test('Renders component without crashing', () => {
    const component = shallow(
      <Container>
        <h1>Some content in here!</h1>
      </Container>
    );

    expect(component).toMatchSnapshot();
  });
});
