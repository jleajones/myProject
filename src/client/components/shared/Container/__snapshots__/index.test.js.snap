// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> Renders component without crashing 1`] = `
<div>
  <h1>
    Some content in here!
  </h1>
</div>
`;
